core/src/test/java/io/grpc/internal/ClientCallImplTest.java

```diff
@@ -383,11 +380,6 @@ public void contextCancellationCancelsStream() throws Exception {
 
     call.start(callListener, new Metadata());
 
-    ClientStreamListener listener = listenerArgumentCaptor.getValue();
```

Comment:
I don't believe it was needed any more.  Github has lost the lines you commented on, so I can't actually see the context for this.